py/picca/delta_extraction/quasar_catalogues/desi_quasar_catalogue.py

```diff
@@ -59,7 +68,11 @@ def __init__(self, config):
         self._parse_config(config)
 
         # read quasar catalogue
-        self.catalogue = self.read_catalogue()
+        self.read_catalogue()
+
+        # if not all surveys are specified, then filter the catalogue
+        if "all" not in self.keep_surveys:
```

Comment:
this needs to be documented somewhere, this would also include special tiles, ie use more than the default

Reply:
I'll change `all` to not include the special tiles.  I'll also add this in the configuration tutorials

Reply:
this should be addressed now